apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -150,6 +151,47 @@ Page {
                 }
                 onClicked: pageStack.push("PermissionPage.qml")
             }
+
+            BrowserListItem {
+                //% "Save destination"
+                label: qsTrId("settings_browser-la-save_destination")
+                iconSource: "image://theme/icon-m-download"
+                value: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Download dir"
+                        return qsTrId("sailfish_browser-me-download_dir")
+                    } else {
+                        //% "Ask"
+                        return qsTrId("sailfish_browser-me-ask")
+                    }
+                }
+
+                secondaryText: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Downloaded files will be saved to %1"
+                        return qsTrId("sailfish_browser-me-will_be_saved_to_download").arg(WebEngineSettings.downloadDir)
```

Comment:
Looking at the design, this should only show the folder below the user's home folder (i.e. "Downloads" rather than "/home/defaultuser/Downloads".

Presumably the file picker should also be restricted to the user's home as well; there shouldn't be any reason to try to save a file below that.